162-docker_container_publish_all_option.yml

```diff
@@ -1,4 +0,0 @@
----
-minor_changes:
```

Comment:
Can you rebase to latest `main` so these 'fake' changes go away? They're making reviewing harder :)

Reply:
This is a real change. I missed moving this to `changelogs/fragments` in #162 so now it's just hanging out in root after 1.8.0.

Reply:
Ooops. I completely missed that...

I'll do the move in a separate commit (and add a note that this was already added in 1.8.0).

Reply:
Thanks, I'll close this and look into an info module instead.